vmdb/lib/workers/worker_base.rb

```diff
@@ -421,7 +421,8 @@ def parent_is_alive?(parent)
   end
 
   def check_parent_process
-    server = MiqServer.my_server(true)
+    # ensure attributes like "last_heartbeat" are up to date
+    server.reload if server
```

Comment:
@blomquisg I think `parent_is_alive?`, called one line down is also not called anywhere else, I think it can be deleted.  :sparkling_heart:   I'll look at git history and confirm.